setup.py

```diff
@@ -0,0 +1,27 @@
+from distutils.core import setup
+setup(
+  name = 'swmr_tools',         # How you named your package folder (MyLib)
+  packages = ['swmr_tools'],   # Chose the same as "name"
+  version = '0.1',      # Start with a small number and increase it with every change you make
+  license='MIT',        # Chose a license from here: https://help.github.com/articles/licensing-a-repository
+  description = 'Python iterator for safely monitoring nexus files',   # Give a short description about your library
+  author = 'Richard E Parke',                   # Type in your name
+  author_email = 'richard.parke@diamond.ac.uk',      # Type in your E-Mail
+  url = 'https://github.com/rparke/Iterator',   # Provide either the link to your github or to your website
```

Comment:
Out of date details

Reply:
Updated details:
- author = Diamond Light Source ltd
- author_email = scientificsoftware@diamond.ac.uk
- url = https://github.com/DiamondLightSource/python-swmrtools